packages/notifications/docs/05-customizing-notifications.md

```diff
@@ -10,6 +10,15 @@ Notifications come fully styled out of the box. However, if you want to apply yo
 
 Notifications have dedicated CSS classes you can hook into to apply your own styling. Open the inspector in your browser to find out which classes you need to target.
 
+## Positioning notifications
+
+You can configure the alignment of the notifications with a service provider, for example adding the below to the boot method on: AppServiceProvider.php would align the notifications top middle of the browser.
+
+```php
+        Notifications::alignment(Alignment::Center);
```

Comment:
Please fix indentation and add imports for the class and enums

Reply:
Sorry Dan, been a bit swamped. Thanks for the cleanup! I was using github online for these.